Extensions/TileMap/JsExtension.js

```diff
@@ -482,6 +494,27 @@ module.exports = {
       );
 
       this._pixiObject = new Tilemap.CompositeRectTileLayer(0);
+
+      // Implement `containsPoint` so that we can set `interactive` to true and
+      // the Tilemap will properly emit events when hovered/clicked.
+      // By default, this is not implemented in pixi-tilemap.
+      this._pixiObject.containsPoint = (position) => {
```

Comment:
Perhaps @ivanpopelyshev  might be interested in having it added to pixi-tilemap itself? Or I suppose there is a good reason for not having any extra stuff like this